openbb_terminal/economy/economy_controller.py

```diff
@@ -1018,7 +1018,6 @@ def call_ycrv(self, other_args: List[str]):
         parser.add_argument(
             "-d",
             "--date",
-            type=valid_date,
```

Comment:
I would keep this in order to ensure it's still a valid date, and then have `ns_parser.date.strftime("%Y-%m-%d")` passing the argument to `fred_view.display_yield_curve`. WDYT?